wetterdienst/ui/core.py

```diff
@@ -110,7 +110,7 @@ def _get_stations_request(
         "start_date": start_date,
         "end_date": end_date,
     }
-    if api.provider == Provider.DWD and api.kind == Kind.FORECAST:
+    if api._provider == Provider.DWD and api._kind == Kind.FORECAST:
```

Comment:
That read better before. Is there a reason to make those "private"?

Reply:
We currently only have those two attributes as none private so i thought just make them all private in the classes.

Reply:
All right. I did not see a reason to make them private, but I didn't have a closer look at the other attributes.